README.md

```diff
@@ -96,5 +96,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/atomic
 The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
 
 
-### Things Not Quite Implemented
-People, Groups, Sets, Blogs
+## Things Not Quite Implemented
+People, Sets
```

Comment:
I think you can take off people because I am guessing that is referring to Staff, and I wonder if sets is possibly referring to rubric sets?